durabletask-azuremanaged/durabletask/azuremanaged/client.py

```diff
@@ -18,7 +20,9 @@ def __init__(self, *,
                  taskhub: str,
                  token_credential: Optional[TokenCredential],
                  secure_channel: bool = True,
-                 default_version: Optional[str] = None):
+                 default_version: Optional[str] = None,
+                 log_handler=None,

```

Comment:
Can you add a typehint please (everywhere where we use `log_handler` and `log_formatter`)

Reply:
Yep done